app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_instances = []
+
+    for person in people:
+        list_of_instances.append(Person(person["name"], person["age"]))
+
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
```

Comment:
use get() method instead of two conditions

Reply:
good advice